packages/neos-ui-ckeditor5-bindings/src/ckEditorApi.js

```diff
@@ -58,26 +58,34 @@ export const createEditor = store => async options => {
     return NeosEditor
         .create(propertyDomNode, ckEditorConfig)
         .then(editor => {
-            const debouncedOnChange = debounce(() => onChange(cleanupContentBeforeCommit(editor.getData())), 1500, {maxWait: 5000});
-            editor.model.document.on('change:data', debouncedOnChange);
-            editor.ui.focusTracker.on('change:isFocused', event => {
-                if (!event.source.isFocused) {
-                    // when another editor is focused commit all possible pending changes
-                    debouncedOnChange.flush();
-                    return
-                }
+            editor.isReadOnly = isReadOnly;
```

Comment:
i think i also attempted to use the `IsReadOnly` flag from ck and that did not work always correctly ... instead in https://github.com/neos/neos-ui/pull/3842 i made it work by just NOT booting the ckeditor when there is no reason too and think we should do the same here? Not booting the ckeditor and still showing an outline or something is imo much more stable.

Reply:
I see your point and feel free to implement this. For me personally I don't have the time and as long as it works like intended I will just keep it like this.